tests/travis/step-script.sh

```diff
@@ -46,3 +46,10 @@ fi
 ./simple_test
 ./simple_test_static
 ./api_tests
+
+if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
+    echo $PWD/libOpenCL.so >clvk.icd
+    export OCL_ICD_VENDORS=$PWD
```

Comment:
We could probably use `OCL_ICD_FILENAMES` to avoid the file. It doesn't matter really.

Reply:
This environment variable is not provided by `ocl-icd`, which I believe is the ICD loader packaged by Debian (and thus what Travis CI is using).

Reply:
Right. We can test with the Khronos loader later (I did use that in my local testing and it worked fine so I'm not overly worried). Most distributions use `ocl-icd` at the moment anyway.